pkg/blobfuse-proxy/install-proxy-rhcos.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -xe
+
+# install blobfuse2
+if [ "${INSTALL_BLOBFUSE}" = "true" ] || [ "${INSTALL_BLOBFUSE2}" = "true" ]
```

Comment:
if we only support blobfuse2 install, then we need to make some changes here for coreos?

https://github.com/kubernetes-sigs/blob-csi-driver/blob/b99e1cf51f7f3dfcccd9dae77c74de5cdab8f6bc/pkg/blobfuse-proxy/server/server.go#L124-L127

Reply:
Nice catch!